src/main/kotlin/graphql/kickstart/tools/SchemaParser.kt

```diff
@@ -168,6 +168,7 @@ class SchemaParser internal constructor(
                     .definition(inputDefinition)
                     .description(getDocumentation(inputDefinition, options))
                     .apply { inputDefinition.defaultValue?.let { v -> defaultValueLiteral(v) } }
+                    .apply { getDeprecated(inputDefinition.directives)?.let { deprecate(it) } }
```

Comment:
Can we also add this to the arguments in `createDirective`?

Reply:
Done, thanks for catching that.